src/imageio.cs

```diff
@@ -1169,4 +1213,26 @@ interface CGCopyImageSourceOptions {
 		IntPtr kOrientation { get; }
 	}
 #endif
+
+	[Mac (10, 13), iOS (11,0), TV (11,0), Watch (4,0)]
+	public enum CGImageAuxiliaryDataType {
+		[Field ("kCGImageAuxiliaryDataTypeDepth")]
+		Depth,
+
+		[Field ("kCGImageAuxiliaryDataTypeDisparity")]
+		Disparity,
+	}
+
+	[Partial]
+	[Mac (10, 13), iOS (11,0), TV (11,0), Watch (4,0)]
+	public interface CGImageAuxiliaryDataInfo {
+		[Field ("kCGImageAuxiliaryDataInfoData")]
+		NSString Data { get; }
+
+		[Field ("kCGImageAuxiliaryDataInfoDataDescription")]
+		NSString DataDescription { get; }
+
+		[Field ("kCGImageAuxiliaryDataInfoMetadata")]
+		NSString kMetadata { get; }
```

Comment:
Ah, I see
```
		[Internal][Field ("kCGImageDestinationMetadata")]
		IntPtr kMetadata { get; }
```
the main difference is that the existing constants are `internal` (not visible) so the name does not matter much.

Your new code should use the same pattern, both `[Internal]` and use `IntPtr` (no need for a managed `NSString` just to hold the address).